Doc/reference/compound_stmts.rst

```diff
@@ -378,8 +382,13 @@ exception group with an empty message string. ::
    ...
    ExceptionGroup('', (BlockingIOError()))
 
-An :keyword:`!except*` clause must have a matching type,
-and this type cannot be a subclass of :exc:`BaseExceptionGroup`.
+Unlike :keyword:`except` clauses, :keyword:`!except*` clauses must have a matching
+expression; ``except*:`` is not valid syntax. When the handler is evaluated, the
+expression must evaluate to a type or a tuple containing types. Each type must
+be a subclass of :exc:`BaseException` and must not be a subclass of
+:exc:`BaseExceptionGroup`; otherwise, the runtime will raise a :exc:`TypeError`
+when the handler is evaluated.
```

Comment:
Can we not repeat this twice? It is confusing because people assume it will somehow be different from the except case.

Reply:
Replaced the duplicate part with "In addition to the requirements for matching expressions in except clauses"

```
In addition to the
requirements for matching expressions in :keyword:`except` clauses, the
types in the matching expression of an :keyword:`!except*` clause cannot
be :exc:`BaseExceptionGroup` or a subclass of :exc:`BaseExceptionGroup`;
otherwise, the runtime will raise a :exc:`TypeError` when the handler is
evaluated.
```